src/Components/Web.JS/src/Platform/WebView/WebViewIpcSender.ts

```diff
@@ -32,11 +32,11 @@ function base64EncodeByteArray(data: Uint8Array) {
   for (let i = 0; i < data.length; i++) {
     charBytes[i] = String.fromCharCode(data[i]);
   }
-  const dataBase64Encoded = btoa(charBytes.join(''));
+  const dataBase64Encoded = Buffer.from(charBytes.join(''), 'base64');
```

Comment:
Are you sure this is correct? Can you confirm it's been tested in WebView?

Reply:
I reverted this, seems like this is only available in NodeJS on the server side! Thanks for the catch!